api/src/main/java/org/openmrs/module/mirebalais/apploader/CustomAppLoaderFactory.java

```diff
@@ -2202,13 +2202,6 @@ private void enableHIVForms() {
                 null,
                 PihEmrConfigConstants.PROGRAM_HIV_UUID + ".includeFragments",
                 map("patientVisitsPage", patientVisitsPageWithSpecificVisitUrl)));
-
-
-
-        extensions.add(cloneAsHivOverallAction(findExtensionById(Extensions.CREATE_VISIT_OVERALL_ACTION)));
-        if (config.isComponentEnabled(Components.MARK_PATIENT_DEAD)) {
-            extensions.add(cloneAsHivOverallAction(findExtensionById(Extensions.MARK_PATIENT_DEAD_OVERALL_ACTION)));
```

Comment:
these are no longer necessary because we are adding them all below, correct?

Reply:
Yes, that is correct.  If we left these in place, we'd have duplicate actions for these things.